src/Resources/Tags/Client/TagsClient.cs

```diff
@@ -66,20 +66,32 @@ public TagsClient()
 
         public List<PSTag> ListTags()
         {
-            var result = ResourceManagementClient.Tags.List();
-            List<PSTag> tags = new List<PSTag>();
-
-            do
+            var page = ResourceManagementClient.Tags.List();
```

Comment:
Should this variable be named `pageTags` to better express its meaning?

Reply:
Thanks. Renamed to `pageOfTags`